packages/calcite-components/src/components/list/list.tsx

```diff
@@ -731,15 +727,23 @@ export class List
     this.updateFilteredData();
   }
 
+  private get effectiveFilterProps(): string[] {
```

Comment:
We should consider extracting a utility for `combobox` once we tackle https://github.com/Esri/calcite-design-system/issues/9830.